jest/jest.config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
   ],
   setupTestFrameworkScriptFile: "<rootDir>/packages/webviz-core/src/test/setupTestFramework.js",
   moduleNameMapper: {
+    "worker-loader!./ChartJSWorker": "<rootDir>/webviz-core/src/test/MockWorker.js",
```

Comment:
Should we do this by default for `worker-loader!*`?

Reply:
Why is this path different from the others around it? (missing `/packages/`)

Reply:
Ah yeah, didn't notice that; that seems wrong! 😮

Reply:
Whoops, didn't notice these comments before merging. Fixed with https://github.com/cruise-automation/webviz/pull/343